app/grandchallenge/components/models.py

```diff
@@ -902,6 +912,11 @@ def _clean_overlay_segments(self):
                 "Overlay segments should only be set for segmentations"
             )
 
+        if not self.overlay_segments_is_contiguous:
```

Comment:
Only saw this now. This now requires all overlay_segments to be contiguous, but we have non-contiguous overlay_segments already in the database.

Reply:
Yes that's fine, they're not new and if they need to be updated will need to add intermediate values.